### ISSUE TITLE
A11y_AzureDataStudio_New deployment - SQL Server on Windows_ Status Messages:Status messages is not announced by Narrator while installing or Canceling

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18598]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG4.1.3; #Win32; #AINotInScope; #Status Messages;  #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2. Launch Azure Data Studio.
3. Select the Connections icon (if not already selected) from left bar.
4. Click on the ellipses '...' and select "New Deployment"
5. In the Deployment options, select "SQL Server on Windows" card.
5. From the dropdown for Version under Options, select from the available options.
6. Click the "Select" button
7. This will start the setup installer for SQL Server setup.
8. Follow the instructions through the setup process to install a SQL Server on Windows.
9. Verify status messages after installing/Canceling is announced by narrator or not

### Actual Result:
Status messages is not announced by Narrator while installing or Canceling

### Expected Result:
Status messages should announce by Narrator while installing or Canceling

### Attachment: 

**Narrator Attachment:** 
![A11y_AzureDataStudio_New deployment - SQL Server on Windows_ Status Messages_Narrator](https://user-images.githubusercontent.com/100189407/156169138-8fe261d0-722e-46a6-88a7-309479c56117.gif)

**NVDA  Attachment:** 
![A11y_AzureDataStudio_New deployment - SQL Server on Windows_ Status Messages_NVDA](https://user-images.githubusercontent.com/100189407/156169516-3d63097c-de26-461c-9b82-727adc12dd25.gif)

**Comment by alanrenmsft** [comment]
the install SQL server feature you tested is not a part of AzureDataStudio. so closing it, let me know if you don't know the owner of the sql server installer.

**Comment by VijayaBharathyA** [comment]
@Matteo-T

As per above comments bug is external to SQL server, could you please let us know the Service tree name, VSO details and Area path to log a bug.

**Comment by VijayaBharathyA** [comment]
#Closed;
### Verified the issue in SQL 2022:
Issue is fixed
![image](https://user-images.githubusercontent.com/100189407/218945524-535f03df-9181-4eae-bfac-38e56bcd1466.png)
![image](https://user-images.githubusercontent.com/100189407/218945629-6d972c0e-d9c0-4e7b-81e6-43eb5a486aa6.png)
